src/Support/Arrayify.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+namespace Deployer\Support;
+
+use Symfony\Component\Console\Input\InputInterface;
+
+final class Arrayify
+{
+    public static function options(InputInterface $input): array
+    {
+        $options = array_filter($input->getOptions());
+
+        return iterator_to_array(
+            \iter\flatten(
```

Comment:
Is is possible to implement without extra dependencies?

Reply:
Yes, of course, and it would be more verbose, inelegant and harder to maintain.

Reply:
Let’s do it)

Reply:
If you want to change this, I'll leave it to you.